CONTRIBUTING.md

```diff
@@ -40,8 +40,9 @@ There are some external applications loading resources from http://sourced.tech/
 
 ### Salesforce Pardot Landing Pages
 
-- [source{d} Engine Product Demo Sign-Up](http://go.sourced.tech/engine)
-- [source{d} Lookout Product Demo Sign-Up](http://go.sourced.tech/lookout)
+- [source{d} CE Download](https://go.sourced.tech/community-edition-download)
+- [source{d} EE Demo Sign-Up](https://sourced.tech/products/enterprise-edition/#trial)
```

Comment:
Since the goal of this section is to mention which Pardot pages hotlinks Landing resources, I think this line should be replaced by:
```
https://go.sourced.tech/enterprise-demo
https://go.sourced.tech/l/578561/2019-04-24/5tqgk
```